airflow/contrib/kubernetes/worker_configuration.py

```diff
@@ -89,11 +100,18 @@ def _get_environment(self):
 
         if self.kube_config.airflow_configmap:
             env['AIRFLOW__CORE__AIRFLOW_HOME'] = self.worker_airflow_home
-        if self.kube_config.worker_dags_folder:
-            env['AIRFLOW__CORE__DAGS_FOLDER'] = self.kube_config.worker_dags_folder
+            env['AIRFLOW__CORE__DAGS_FOLDER'] = self.worker_airflow_dags
```

Comment:
this always seemed like such an issue that it's hard coded in the `master` branch, I'm glad this is being addressed